src/pytest_bdd/parser.py

```diff
@@ -171,6 +171,8 @@ def parse_feature(basedir: str, filename: str, encoding: str = "utf-8") -> Featu
         elif mode == types.EXAMPLES:
             mode = types.EXAMPLES_HEADERS
             scenario.examples.line_number = line_number
+            scenario.examples.tags = get_tags(prev_line)
+            scenario.examples.examples
```

Comment:
I'm not clear on what this second line is doing.

Reply:
Same

Reply:
Oops. A late night unfinished thought